test/built-ins/Temporal/Instant/prototype/round/roundingincrement-non-integer.js

```diff
@@ -5,11 +5,11 @@
 esid: sec-temporal.instant.prototype.round
 description: Rounding for roundingIncrement option
 info: |
-    sec-temporal-totemporalroundingincrement step 7:
-      7. Set _increment_ to floor(ℝ(_increment_)).
+    sec-temporal-totemporalroundingincrement:
+      3. Let _integerIncrement_ be truncate(ℝ(_increment_)).
 features: [Temporal]
 ---*/
 
 const instant = new Temporal.Instant(1_000_000_000_000_000_005n);
 const result = instant.round({ smallestUnit: "nanosecond", roundingIncrement: 2.5 });
```

Comment:
These tests implicitly rely upon the "halfExpand" default, which is not obvious and could be implemented incorrectly independent of `round`.
```suggestion
const instant = new Temporal.Instant(1_000_000_000_000_000_005n);
const result = instant.round({ smallestUnit: "nanosecond", roundingMode: "expand", roundingIncrement: 2.5 });
```